# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1262675107 18000
#      Tue Jan 05 02:05:07 2010 -0500
# Node ID 7494d047e98d67fa3a16f664f8a7885a7ee6b82b
# Parent  ec0c445d0d6f1f21021b97717f181a021f9ec98b
bootstrap.conf: remove nonexistent "fstat" module

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-05  John W. Eaton  <jwe@octave.org>
+
+	* bootstrap.conf (gnulib_modules): Remove nonexistent "fstat" module.
+
 2009-12-22  Rik <octave@nomad.inbox5.com>
 
 	* WWW/: remove web directory no longer used
 
 2010-01-04  Jaroslav Hajek  <highegg@gmail.com>
 
 	* NEWS: Update.
 
diff --git a/bootstrap.conf b/bootstrap.conf
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -15,17 +15,16 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 # gnulib modules used by this package.
 gnulib_modules="
   crypto/md5
   fnmatch
-  fstat
   glob
   lstat
   mkdir
   stat
   strftime
 "
 
 # Additional xgettext options to use.  Use "\\\newline" to break lines.
